src/ape_plugins/_cli.py

```diff
@@ -22,6 +22,19 @@ def cli():
     """
 
 
+def _remove_prefix_add_version(set_convert, _prefix, version=None):
+    set_print = set()
```

Comment:
`set_print` sounds like a method name.
Maybe call this `output` or something?

Reply:
yes, let make it more concise

Reply:
It is less about being concise and more about naming conventions!

If you have a spare moment, I highly the teachings of Uncle Bob. Here is the relevant blog set for variable names: https://thecoderoad.blog/2020/03/29/clean-code-naming-conventions/#:~:text=According%20to%20Uncle%20Bob%20in,%E2%80%9Cintention%2Drevealing%E2%80%9D%20names.

Reply:
Basically, avoid using verbs unless you are naming methods.
`set_print` is like 2 verbs, so in my mind, `set_print` is a method that sets a variable named `print` or something

Reply:
I understand now. Thank you for the clarification and ill add that website to look it. Thank you